docs/user.md

```diff
@@ -254,6 +254,18 @@ metadata:
 Note that timezone is required for `timestamp`. Otherwise, offset is relative
 to UTC, see [RFC 3339 section 5.6) 3339 section 5.6](https://www.ietf.org/rfc/rfc3339.txt).
 
+## Setting up a standby cluster
+
+Standby clusters are like normal cluster but they are streaming from a remote master. Patroni supports it, [read this](https://github.com/zalando/patroni/blob/bd2c54581abb42a7d3a3da551edf0b8732eefd27/docs/replica_bootstrap.rst#standby-cluster) to know more about them. Currently operator supports setting up standby only through an S3 bucket. For this you need to add a section standby in the YAML file as follows.
```

Comment:
> Currently operator supports setting up standby only through an S3 bucket.

This part needs more details. Earlier it is mentioned that a standby cluster streams from some remote cluster and then suddenly we are talking about an S3 bucket. For people not that familiar with our particular setup it may sound confusing.